portfolio/src/main/java/com/google/sps/servlets/ServerCommentsServlet.java

```diff
@@ -0,0 +1,59 @@
+package com.google.sps.servlets;
+
+import com.google.gson.Gson;
+import com.google.sps.data.ReviewerData;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+@WebServlet("/comments")
+public final class ServerCommentsServlet extends HttpServlet {
+
+  private final List<ReviewerData> reviews = new ArrayList<>();
```

Comment:
Please apply the rules for whitespace https://google.github.io/styleguide/javaguide.html